### PR TITLE
add neuronswap

### PR DESCRIPTION
**Pull request by byeonggoon**
##### Twitter Link:
https://twitter.com/NEURONswap

##### List of audit links if any:
x

##### Website Link:
https://www.neuronswap.com/

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):

neuron token

##### Current TVL:
$4,329,305

##### Chain:
klaytn

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
x

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
x

##### Short Description (to be shown on DefiLlama):
NEURONswap is a decentralized exchange (DEX) that has a goal to popularize blockchain-based decentralized financial applications, decentralized finance and revolutionize the NFT minting system.

##### Token address and ticker if any:
klaytn network
token address = 0x340073962a8561cb9e0c271aab7e182d5f5af5c8
ticker = NR 

##### Category (full list at https://defillama.com/categories) *Please choose only one:
[Dexes]

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
x

##### forkedFrom (Does your project originate from another project):
Uniswap ,SushiSwap

##### methodology (what is being counted as tvl, how is tvl being calculated):
Tvl counts the tokens locked on AMM pools and staking counts the NR that has been staked. Data is pulled from the 'https://core.neuronswap.com/api/dashboard

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/neuronswap/index.js exports TVL: 
        
 
 
    klaytn                    4.21 M
    
    total                    4.21 M

**Comment by waynebruce0x** [comment]
Whats counted in data.totalValueLocked? Is the API open source?

**Comment by byeonggoon** [comment]
totalValueLocked = totalValueLocked = total lp value + total staking NR . 

Should the API be open source? The API can be called at https://core.neuronswap.com/api/dashboard.

**Comment by waynebruce0x** [comment]
where can I stake NR on the dapp UI? Can staked NR be separated out and exported as 'staking'?

**Comment by byeonggoon** [comment]
https://www.neuronswap.com/staking -> 'Total NR Staked' is staked NR on the dapp UI ,

It takes some time to get the staked NR. I will comment again.

**Comment by waynebruce0x** [comment]
The 'tvl' export shouldn't include staked NR, with staked NR exported as 'staking' instead - Can that be done?

**Comment by byeonggoon** [comment]
The code was updated. In the same api, lp tvl and staking nr were calculated separately. please confirm

**Comment by llamatester** [comment]
Error while running adapter at projects/neuronswap/index.js: 
        
 
 
    
        Please move the following keys into the chain: staking
    
        We have a new adapter export specification now where tvl and other chain specific information are moved inside chain export.
        For example if your protocol is on ethereum and has tvl and pool2, the export file would look like:
        
            module.exports = {
              methodlogy: '...',
              ethereum: {
                tvl: 
                pool2:
              }
            }

File: projects/neuronswap/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+module.exports = {
+  methodology: `Tvl counts the tokens locked on AMM pools and staking counts the NR that has been staked. Data is pulled from the 'https://core.neuronswap.com/api/dashboard'`,
+  klaytn: {
+    fetch: NRtvl,
+    staking: stakedNR,
+  },
+
+  fetch: async () => await NRtvl(),
+  staking: async () => await stakedNR(),
```
Comment (waynebruce0x) [inline projects/neuronswap/index.js:27]: this line can be deleted

**Comment by byeonggoon** [comment]
The code was updated. please confirm.! 

Thank you for your kind guidance.

**Comment by llamatester** [comment]
Error while running adapter at projects/neuronswap/index.js: 
        
 
  
    
    
    Error: invalid BigNumber string (argument="value", value=".", code=INVALID_ARGUMENT, version=bignumber/5.6.0)
        at Logger.makeError (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@ethersproject/logger/lib/index.js:233:21)
        at Logger.throwError (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@ethersproject/logger/lib/index.js:242:20)
        at Logger.throwArgumentError (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@ethersproject/logger/lib/index.js:245:21)
        at Function.BigNumber.from (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@ethersproject/bignumber/lib/bignumber.js:201:27)
        at Object.sumSingleBalance (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/generalUtil.js:80:62)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/computeTVL/index.js:245:43
        at step (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/computeTVL/index.js:33:23)
        at Object.next (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/computeTVL/index.js:14:53)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/computeTVL/index.js:8:71
        at new Promise (<anonymous>) {
      reason: 'invalid BigNumber string',
      code: 'INVALID_ARGUMENT',
      argument: 'value',
      value: '.'
    }

**Comment by byeonggoon** [comment]
hi sir 


When you run neuronswap/index.js, NRtvl and stakedNR appear, and the values are "824194.99", "3308128.46" and so on. But why am I getting bignumber error? Can you explain me?

**Comment by byeonggoon** [comment]
스크린샷 2022-04-18 오후 12 02 00

please update token name is Neuron ( not Neuronswap :)) thx.

**Pull request closed without merging**